### ISSUE TITLE
Cannot lock backup dir `'

### ISSUE THREAD

**Issue by ckauhaus** [ckauhaus/backy-extract#1]
backy-extract fails when invoked from the backup dir as pwd, 

```
# backy-extract JBs7pis8kJgtZiT2WPuD4W >/dev/null
Error:  Cannot lock backup dir `'
Detail: Resource temporarily unavailable (os error 11)
```

**Comment by ckauhaus** [comment]
Related problem: creates spurious `.purge` files when called with just a directory as only argument.

**Comment by ckauhaus** [comment]
Fixed in df9c5ff6afff0b13103a4c3d0a1778fc3d2c364e
